### PR TITLE
Fix --source option

### PR DESCRIPTION
**Pull request by Luc45**
This PR fixes the `--source` option on `run:e2e` as discussed in Slack.

### CODE REVIEW

**Comment by MrJnrman** [comment]
Thanks for working on this @Luc45. It works great with absolute paths, but not with relative paths. Maybe we can pass the source through `realpath` when setting up the volumes here https://github.com/woocommerce/qit-cli/blob/9d76f3006ac58a4a908e790d58a46097e303b3ff/src/src/Environment/ExtensionDownload/ExtensionDownloader.php#L92

File: src/src/Commands/CustomTests/RunE2ECommand.php [hunk 1]
```diff
@@ -273,6 +273,7 @@ protected function execute( InputInterface $input, OutputInterface $output ): in
 
 		putenv( 'QIT_UP_AND_TEST=1' ); // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.runtime_configuration_putenv
 		putenv( sprintf( 'QIT_SUT=%s', $woo_extension ) ); // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.runtime_configuration_putenv
+		putenv( sprintf( 'QIT_SUT_SOURCE=%s', $source ) ); // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.runtime_configuration_putenv
```
Comment (Luc45) [inline src/src/Commands/CustomTests/RunE2ECommand.php:285]: @MrJnrman what about we add realpath here?

Comment (MrJnrman) [reply src/src/Commands/CustomTests/RunE2ECommand.php:285]: That'd do the trick

Comment (Luc45) [inline src/src/Commands/CustomTests/RunE2ECommand.php:276]:
```suggestion
		putenv( sprintf( 'QIT_SUT_SOURCE=%s', realpath( $source ) ) ); // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.runtime_configuration_putenv
```

**Review by MrJnrman — APPROVED** [review]
Works great with themes and plugins. Thanks Lucas! 🚢

**Pull request merged**